router/batchrouter/batchrouter_isolation_test.go

```diff
@@ -258,7 +259,7 @@ func BatchrouterIsolationScenario(t testing.TB, spec *BrtIsolationScenarioSpec)
 	config.Set("RUDDER_TMPDIR", os.TempDir())
 
 	t.Logf("Seeding batch_rt jobsdb with jobs")
-	m.seedBrtDB(t, spec)
+	m.seedBrtDB(t, spec, postgresContainer.DB)
```

Comment:
I don't think this makes much of a difference here, since we are messing with the global config in this test anyway. Changing a very small part of this test wouldn't make it behave better.